acceptance/framework/config/config.go

```diff
@@ -22,16 +22,67 @@ const (
 	LicenseSecretKey  = "key"
 )
 
-// TestConfig holds configuration for the test suite.
-type TestConfig struct {
-	Kubeconfig    string
+type KubeEnv struct {
+	KubeConfig    string
 	KubeContext   string
 	KubeNamespace string
+}
+
+func KubeEnvListFromStringList(kubeConfigs, kubeContexts, kubeNamespaces []string) []KubeEnv {
+	// Determine the longest list length
+	maxLen := 0
+	lenConf := len(kubeConfigs)
+	if maxLen < lenConf {
+		maxLen = lenConf
+	}
+
+	lenCtx := len(kubeContexts)
+	if maxLen < lenCtx {
+		maxLen = lenCtx
+	}
 
-	EnableMultiCluster     bool
-	SecondaryKubeconfig    string
-	SecondaryKubeContext   string
-	SecondaryKubeNamespace string
+	lenNs := len(kubeNamespaces)
+	if maxLen < lenNs {
+		maxLen = lenNs
+	}
+
+	// If all are empty, then return a single empty entry
+	out := make([]KubeEnv, 0)
+	if maxLen == 0 {
+		out = append(out, KubeEnv{})
+		return out
+	}
+
+	// Pad lists if required, all list should be made the same length
```

Comment:
That is correct, if you wanted config3 paired with ns3 and no ns2 it would be 
```
configs: config1, config2, config3
context: context1, context2
namespaces: ns1, "", ns2
```
```
config1, context1, ns1
config2, context2, ""
config3, "", ns3
```

It's kind funky with the command line inputs

Reply:
A bunch of permutations are handled in the unit test